src/LibraryViewExtensionWebView2/web/library/build/librarie.min.js.LICENSE.txt

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
I've seen librarie.min.js needs to be included (I suppose as it is an EmbeddedResource ) otherwise we'll get an error in Library controller related to `StreamReader`. @RobertGlobant20 do you why this might be happening?

Reply:
@Enzo707 have you resolved this or do you still need unblocking here?